guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -0,0 +1,30 @@
+[id="disabling-cockpit-on-project_{context}"]
+= Disabling {CockpitTitle} on {Project}
+
+Perform the following procedure if you want to disable {Cockpit} on {Project}.
+
+.Procedure
+. Disable {Cockpit} with the `{foreman-installer}` command:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --no-enable-foreman-plugin-remote-execution-cockpit
+----
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Remote execution*.
+. Erase the value of the *Cockpit URL* setting.
```

Comment:
Does this even matter?

Reply:
Sadly, yes. Some UI elements appear only when it is set

Reply:
The procedure was taken from https://bugzilla.redhat.com/show_bug.cgi?id=2168503